packages/eas-cli/src/project/remoteVersionSource.ts

```diff
@@ -12,21 +12,25 @@ export async function ensureVersionSourceIsRemoteAsync(
   { nonInteractive }: { nonInteractive: boolean }
 ): Promise<void> {
   const easJsonCliConfig = await EasJsonUtils.getCliConfigAsync(easJsonAccessor);
-  if (easJsonCliConfig?.appVersionSource === AppVersionSource.REMOTE) {
+  if (easJsonCliConfig?.appVersionSource !== AppVersionSource.LOCAL) {
     return;
   }
   if (nonInteractive) {
     throw new Error(
       `This project is not configured for using remote version source. Add ${chalk.bold(
         '{"cli": { "appVersionSource": "remote" }}'
-      )} in eas.json or re-run this command without "--non-interactive" flag.`
+      )} in eas.json (or remove ${chalk.bold(
+        '{"cli": { "appVersionSource": "local" }}'
+      )} to use "remote" as a default) or re-run this command without "--non-interactive" flag.`
```

Comment:
```suggestion
      )} in eas.json or re-run this command without the "--non-interactive" flag.`
```

It doesn't seem necessary. Overwhelming users with too much information might confuse them.